Mining.md

```diff
@@ -104,29 +104,11 @@ The **minimum** requirements for Zilliqa mining nodes are:
 
 > **NOTE:** If you are using a home router, you are most probably in a NAT environment.
 
-If you are in NAT environment, you can either:
-
-- Do single port forwarding using **Option 1a**. This should be your **DEFAULT OPTION**.
-- Enable UPnP mode using **Option 1b** if your router does support UPnP.
-
 If you have a public IP address, you can skip this network setup entirely.
 
-- **(Option 1a)** Port forward to port `33133` for both external port (port range) and internal port (local port). You will also have to select the option for **BOTH** TCP and UDP protocol in your router menu when port forwarding. <br><br> An example of this process can be found [**HERE**](https://www.linksys.com/us/support-article?articleNum=136711). After port forwarding, you may check if you have successfully port forwarded with this [**Open Port Check Tool**](https://www.yougetsignal.com/tools/open-ports/).
-
-- **(Option 1b)** Enable UPnP mode on your home router. Please Google how to access your home router setting to enable UPnP, an example can be found [**HERE**](https://routerguide.net/how-to-enable-upnp-for-rt-ac66u/). You can check if you have successfully enabled UPnP by installing the following tool:
-    ```shell
-    sudo apt-get install miniupnpc
-    ```
-   Then type the following in the command line:
-    ```shell
-    upnpc -s
-    ```
-   You should get a message showing either:
-
-     - "List of UPNP devices found on the network : ..."
-     - **OR** "No IGD UPnP Device found on the network !".
+If you are in NAT environment, you will need to
```

Comment:
I think we also not require this part.
![image](https://user-images.githubusercontent.com/66236813/133401671-7bcd8865-625a-4bf7-95c0-05e94ec5e737.png)

Reply:
So it is not possible to be behind a router and do port forwarding with a router having a public IP address anymore?

Reply:
Hmm. I think it should work ideally, as static public IP is not meant for miner nodes.
Also even if public IP changes which In case happens for home router, mining should not be a problem.
We can keep it.